cmake/Modules/FindUDev.cmake

```diff
@@ -9,45 +9,45 @@
 # Redistribution and use of this file is allowed according to the terms of the BSD license.
```

Comment:
Not sure if we can/should modify this file

Reply:
My impression was that BSD licenses allowed modification but I'm no software license expert.